docs/src/components/ui/DesktopMenu.tsx

```diff
@@ -134,12 +135,161 @@ const SocialLinksItem = styled.li`
   }
 `;
 
+const MenuItemDropDownWrapper = styled.div`
+  width: 494px;
+  height: 200px;
+  position: absolute;
+  top: 25px;
+  left: 50%;
+  transform: translateX(-50%);
+`;
+
+const MenuItemDropdown = styled.div`
+  --bg-color: ${tm(({ colors }) => colors.neutral0)};
+  width: 494px;
+  height: 176px;
+  box-shadow: 0px 9px 28px 8px rgba(0, 0, 0, 0.05);
+  filter: drop-shadow(0px 6px 50px rgba(10, 11, 13, 0.41));
+  border-radius: 4px;
+  background-color: var(--bg-color);
```

Comment:
Why can't we set bg color directly as we did it everywhere before? If the reason that we need this in the child element - can't we use `inherit` value instead (see a comment below)

Reply:
Corrected